src/asim/configs/resident/summarize_preprocessor.csv

```diff
@@ -0,0 +1,434 @@
+,_tncin_loc_wlk_xwait,trip_odt_skims['TNCIN_LOC_XFERWAIT']
+,_tncin_mix_wlk_xwait,trip_odt_skims['TNCIN_MIX_XFERWAIT']
+,_tncin_prm_wlk_xwait,trip_odt_skims['TNCIN_PRM_XFERWAIT']
+,_knrout_loc_wlk_xwait,trip_odt_skims['KNROUT_LOC_XFERWAIT']
+,_knrout_mix_wlk_xwait,trip_odt_skims['KNROUT_MIX_XFERWAIT']
+,_knrout_prm_wlk_xwait,trip_odt_skims['KNROUT_PRM_XFERWAIT']
+,_pnrout_loc_wlk_xwait,trip_odt_skims['PNROUT_LOC_XFERWAIT']
+,_pnrout_mix_wlk_xwait,trip_odt_skims['PNROUT_MIX_XFERWAIT']
+,_pnrout_prm_wlk_xwait,trip_odt_skims['PNROUT_PRM_XFERWAIT']
+,_tncout_loc_wlk_xwait,trip_odt_skims['TNCOUT_LOC_XFERWAIT']
+,_tncout_mix_wlk_xwait,trip_odt_skims['TNCOUT_MIX_XFERWAIT']
+,_tncout_prm_wlk_xwait,trip_odt_skims['TNCOUT_PRM_XFERWAIT']
+,_walk_loc_wlk_xwait,trip_odt_skims['WALK_LOC_XFERWAIT']
+,_walk_mix_wlk_xwait,trip_odt_skims['WALK_MIX_XFERWAIT']
+,_walk_prm_wlk_xwait,trip_odt_skims['WALK_PRM_XFERWAIT']
+
+# Walk access time (WACC) (for all transit modes)
+,_wlk_wacc,_SHORT_WALK / _WALK_SPEED * 60 # Constant assumption of short walk access
+
+# Auxiliary walk time (WAUX)
+#####,_drv_com_wlk_waux,trip_odt_skims['DRV_COM_WLK_WAUX']/100
+#####,_drv_exp_wlk_waux,trip_odt_skims['DRV_EXP_WLK_WAUX']/100
+#####,_drv_hvy_wlk_waux,trip_odt_skims['DRV_HVY_WLK_WAUX']/100
+#####,_drv_loc_wlk_waux,trip_odt_skims['DRV_LOC_WLK_WAUX']/100
+#####,_drv_lrf_wlk_waux,trip_odt_skims['DRV_LRF_WLK_WAUX']/100
+#####,_wlk_com_wlk_waux,trip_odt_skims['WLK_COM_WLK_WAUX']/100
+#####,_wlk_exp_wlk_waux,trip_odt_skims['WLK_EXP_WLK_WAUX']/100
+#####,_wlk_hvy_wlk_waux,trip_odt_skims['WLK_HVY_WLK_WAUX']/100
+#####,_wlk_loc_wlk_waux,trip_odt_skims['WLK_LOC_WLK_WAUX']/100
+#####,_wlk_lrf_wlk_waux,trip_odt_skims['WLK_LRF_WLK_WAUX']/100
+
+# Walk egress time (WEGR) (for all transit modes)
+,_wlk_wegr,_SHORT_WALK / _WALK_SPEED * 60 # Constant assumption of short walk egress
+
+# Drive access/egress time - ACC column used bc https://github.com/sandag/abm/wiki/output-files#transit_skimsomx
+##### SHOULD THIS INCLUDE _EGR column as well?
+,_knrin_loc_wlk_acc,trip_odt_skims['KNRIN_LOC_ACC']
+,_knrin_mix_wlk_acc,trip_odt_skims['KNRIN_MIX_ACC']
+,_knrin_prm_wlk_acc,trip_odt_skims['KNRIN_PRM_ACC']
+,_pnrin_loc_wlk_acc,trip_odt_skims['PNRIN_LOC_ACC']
+,_pnrin_mix_wlk_acc,trip_odt_skims['PNRIN_MIX_ACC']
+,_pnrin_prm_wlk_acc,trip_odt_skims['PNRIN_PRM_ACC']
+,_tncin_loc_wlk_acc,trip_odt_skims['TNCIN_LOC_ACC']
+,_tncin_mix_wlk_acc,trip_odt_skims['TNCIN_MIX_ACC']
+,_tncin_prm_wlk_acc,trip_odt_skims['TNCIN_PRM_ACC']
+,_knrout_loc_wlk_acc,trip_odt_skims['KNROUT_LOC_ACC']
+,_knrout_mix_wlk_acc,trip_odt_skims['KNROUT_MIX_ACC']
+,_knrout_prm_wlk_acc,trip_odt_skims['KNROUT_PRM_ACC']
+,_pnrout_loc_wlk_acc,trip_odt_skims['PNROUT_LOC_ACC']
+,_pnrout_mix_wlk_acc,trip_odt_skims['PNROUT_MIX_ACC']
+,_pnrout_prm_wlk_acc,trip_odt_skims['PNROUT_PRM_ACC']
+,_tncout_loc_wlk_acc,trip_odt_skims['TNCOUT_LOC_ACC']
+,_tncout_mix_wlk_acc,trip_odt_skims['TNCOUT_MIX_ACC']
+,_tncout_prm_wlk_acc,trip_odt_skims['TNCOUT_PRM_ACC']
```

Comment:
There is no 'TNCOUT_LOC_ACC', 'TNCOUT_MIX_ACC', 'TNCOUT_PRM_ACC' in transit skims